src/components/views/post-list.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import './post-list.scss';
+
+class PostList extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const { posts } = this.props;
+        return (
+            <div>{posts.map(post => {
+                let date = new Date(post.date * 1000);
+
+                return (
+                    <article key={post.id} className="article-body">
```

Comment:
Extract Article as a separate component. There could be even 2 of them: for full article and for article preview.